monGARS/core/cortex/curiosity_engine.py

```
@@ -808,3 +902,74 @@ def _summarise_documents(self, documents: Sequence[Mapping[str, Any]]) -> str:
+        try:
+            document = await asyncio.wait_for(
+                self._iris.fetch_document(hit.url),
+                timeout=10,
+            )
+        except asyncio.TimeoutError:
+            logger.debug(
+                "curiosity.iris_document_fetch_timeout",
+                extra={"url": hit.url},
+                exc_info=True,
+            )
+        except Exception:  # pragma: no cover - network dependent
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)